easybuild/easyblocks/t/tensorflow.py

```diff
@@ -99,10 +100,18 @@ def configure_step(self):
 
         tmpdir = tempfile.mkdtemp(suffix='-bazel-configure')
 
+        # filter out paths from CPATH and LIBRARY_PATH
+        package_filter = self.cfg['package_filter']
+        for var in ['CPATH', 'LIBRARY_PATH']:
+            path = os.getenv(var).split(':')
+            filtered_path = [p for fil in package_filter for p in path if fil not in p]
```

Comment:
@damianam Please log the value of the environment variable before and after filtering.